libsoftwarecontainer/src/gateway/iptableentry.h

```diff
@@ -67,6 +67,18 @@ class IPTableEntry
      */
     ReturnCode applyRules();
 
+    /**
+     * @brief Interprets a rule to iptables applicable string
+     * @return string indicating interpreted rule
+     */
+    std::string interpretRule(Rule rule);
+
+    /**
+     * @brief Interprets defaultTarget rule to iptables applicable string
```

Comment:
defaultTarget indicates what happens to packets if they don't match to any rules. 
iptables apply this functionality with setting policy. So defaultTarget here represents default behaviour of packets in NetworkGateway configuration.